database.php

```diff
@@ -0,0 +1,37 @@
+<?php
+function getDb(){
```

Comment:
needs doc blocks for all functions, also type hinting for any parameters and return type hinting. (hint the type hint for db is PDO)